tests/common.go

```diff
@@ -1240,6 +1246,45 @@ func ValidateMountOptionsWithPureVolumes(ctx *scheduler.Context, errChan ...*cha
 
 }
 
+// ValidateCreateOptionsWithPureVolumes is the ginkgo spec for executing file system options check for the given volume
+func ValidateCreateOptionsWithPureVolumes(ctx *scheduler.Context, errChan ...*chan error) {
+	var requiredXfsCreateoption = "-b 2048"
+	var requiredExt4Createoption = "-b size=2048"
+	vols, err := Inst().S.GetVolumes(ctx)
+	log.FailOnError(err, "Failed to get app %s's volumes", ctx.App.Key)
+	log.Infof("volumes of app %s are %s", ctx.App.Key, vols)
+	for _, v := range vols {
+		pvcObj, err := k8sCore.GetPersistentVolumeClaim(v.Name, v.Namespace)
+		if err != nil {
+			err = fmt.Errorf("Failed to get pvc for volume %s. Err: %v", v, err)
+			processError(err, errChan...)
+		}
+
+		sc, err := k8sCore.GetStorageClassForPVC(pvcObj)
+		if err != nil {
+			err = fmt.Errorf("Error Occured while getting storage class for pvc %s. Err: %v", pvcObj, err)
+			processError(err, errChan...)
+		}
+
+		attachedNode, err := Inst().V.GetNodeForVolume(v, defaultCmdTimeout*3, defaultCmdRetryInterval)
+		if err != nil {
+			err = fmt.Errorf("Failed to get app %s's attachednode. Err: %v", ctx.App.Key, err)
+			processError(err, errChan...)
+		}
+		if strings.Contains(fmt.Sprint(sc.Parameters), requiredXfsCreateoption) || strings.Contains(fmt.Sprint(sc.Parameters), requiredExt4Createoption) {
```

Comment:
Is there a way we can make this check more generic? At least I'm thinking search for just "-b " instead of the specific xfs/ext4 options

Reply:
Done.